src/utils/country.utils.ts

```diff
@@ -0,0 +1,18 @@
+import Cookies from "js-cookie";
+import { cloudflareTrace } from "../constants/url.constants";
+
+interface TraceData {
+    loc?: string;
+}
+export const getCountry = async (): Promise<string> => {
```

Comment:
Could u pls add JSDOCS for this function + usage examples 🙏 
<img width="1247" alt="Screenshot 2024-11-06 at 5 33 57 PM" src="https://github.com/user-attachments/assets/88ed3e11-9e8c-4887-8ccd-79d5543e892d">